graphql/types/Product.graphql

```diff
@@ -141,6 +141,14 @@ type Offer {
   Tax: Float
   CacheVersionUsedToCallCheckout: String
   DeliverySlaSamples: [DeliverySlaSamples]
+  """List of discount highlights"""
+  discountHighLights: [Discount]
```

Comment:
you should add `!` after type name to ensure that discount cannot be null (the list must be empty) 

```suggestion
  discountHighLights: [Discount!]
```